apps/client/src/features/app-settings/panel/project-panel/RenameProjectForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { FormControl, Input, Button } from '@chakra-ui/react';
+import { useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+
+import style from './ProjectPanel.module.scss';
+
+export type RenameProjectFormValues = {
+  filename: string;
+};
+
+interface RenameProjectFormProps {
+  filename: string;
+  onSubmit: (values: RenameProjectFormValues) => Promise<void>;
+  onCancel: () => void;
+  submitError: string | null;
+}
+
+export default function RenameProjectForm({ filename, onSubmit, onCancel, submitError }: RenameProjectFormProps) {
+  const {
+    handleSubmit,
+    register,
+    formState: { isSubmitting, isDirty, isValid },
+    setFocus,
+  } = useForm<RenameProjectFormValues>({
+    defaultValues: { filename },
+    values: { filename },
+    resetOptions: {
+      keepDirtyValues: true,
+    },
+  });
+
+  useEffect(() => {
+    setFocus('filename');
+  }, []);
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit(onSubmit)} className={style.form}>
+        <FormControl className={style.formControl}>
+          <Input defaultValue={filename} size='sm' type='text' variant='ontime-filled' {...register('filename')} />
+        </FormControl>
+        <div className={style.actionButtons}>
+          <Button
+            disabled={!isDirty || !isValid || isSubmitting}
+            aria-label='Cancel duplicate project name'
```

Comment:
todo: maybe this is a left over from copy paste. 

question: Does it mean that the interface is so similar that it should be the same? I dont think that showing the two inputs in duplicate gives that much value.
How about prepopulating the fields with the name and only activating the action "Rename" and "Duplicate" once the text is differrent?


Also: we dont really have requirements for accessibility this way, the button label provides adequate accessibility for our e2e tests

Reply:
I agree the two inputs for duplicate might not have as much value, but I'd still like something to show the user the project they are duplicating

should we have a single input that is prepopulated with the duplicated project name and the `Duplicate` button then? I think that makes sense

Reply:
I made duplicate single input

![image](https://github.com/cpvalente/ontime/assets/63670689/414a57e7-b38f-4068-8ec3-ad74dc2ef3e9)